changelogs/unreleased/5043-yangyy93-minor.md

```diff
@@ -0,0 +1,14 @@
+## Add Tracing Support
```

Comment:
This file needs to be renamed to
```
./changelogs/unreleased/5043-yangyy93-major.md
```

Reply:
Yes, this feature should belong to the major, I will update the changelog and add the corresponding documentation

Reply:
This still needs to be changed to major